### DISCUSSION TITLE
Please Add Sub-Folder Batch Upscale

### DISCUSSION THREAD

**Discussion by duelmas** [upscayl/upscayl#949] [Ideas]
Can we help with this?
I upscale video frames for clips and having a way to Batch one folder with the sub folders would be helpful because each clip is separate animations. Happy to help if possible, or open source any art assets on these projects we do if of interest.

**Comment by aaronliu0130** [comment]
https://github.com/upscayl/upscayl/issues/283
Yes, you can help by making a PR for us. Life caught up, unfortunately.

**Reply by duelmas** [reply]
I know html/css mostly but found this... maybe my first PR and not sure it will work but perhaps it's a start... ?

https://www.perplexity.ai/search/can-you-review-the-full-code-f-M4cnzjXkS52eIRKVeCsHnQ

**Comment by duelmas** [comment]
okay, got a start here maybe, idk https://github.com/upscayl/upscayl/commit/52a474b34556c7d62d258e978485db585a46ef23

**Reply by tncicy** [reply]
Sorry, but how do I use this? I just started using this website.

**Reply by aaronliu0130** [reply]
@tncicy What do you mean? This is currently a desktop app.
